spec/jira/resource/agile_spec.rb

```diff
@@ -1,7 +1,11 @@
 require 'spec_helper'
 
 describe JIRA::Resource::Agile do
-  let(:client) { double(options: {rest_base_path: '/jira/rest/api/2', context_path: '/jira'}) }
+  let(:client) do
+    client = double(options: {rest_base_path: '/jira/rest/api/2', context_path: '/jira'})
+    allow(client).to receive(:Issue).and_return(JIRA::Resource::IssueFactory.new(client))
```

Comment:
If i dont akkiw the IssueFactory i cant access the IssueFactory and get a exception which says that :Issue is not available on client. I got this line from the filter_spec.rb:+6 and after insertion the :Issue worked on the client.

This is the exception i get
`RSpec::Mocks::MockExpectationError: #<Double (anonymous)> received unexpected message :Issue with (no args)
./lib/jira/resource/agile.rb:31:in `get_board_issues'
./spec/jira/resource/agile_spec.rb:37:in `block (3 levels) in <top (required)>'
-e:1:in `load'
-e:1:in `<main>'`